doc/build.rst

```diff
@@ -317,51 +290,33 @@ simplest way to install the R package after obtaining the source code is:
   cd R-package
   R CMD INSTALL .
 
-As an alternative, the package can also be loaded through ``devtools::load_all()``
-from the same subfolder ``R-package`` in the repository's root, and by extension, can
-be installed through RStudio's build panel if one adds that folder ``R-package`` as an
-R package project in the RStudio IDE.
+Use the environment variable ``MAKEFLAGS=-j$(nproc)`` if you want to speedup the build. As
```

Comment:
This doesn't work for R packages. `MAKEFLAGS` needs to be in the user file `~/.R/Makevars`.

Reply:
I'm running this command at the moment with Ubuntu 22.04,  and R 4.4.2, it's effective.

Reply:
`MAKEFLAGS=-j$(nproc) R CMD INSTALL`

Reply:
That's curious (doesn't work for me with `install.packages`). But would that work if you use devtools to install?

Reply:
Yes, it works.

``` R
devtools::load_all(path = "/home/jiamingy/workspace/xgboost_dev/xgboost/R-package")
```

``` bash
MAKEFLAGS=-j$(nproc) Rscript ./basic.R
```